src/Faker/Provider/el_GR/Company.php

```diff
@@ -72,6 +72,14 @@ public static function doy()
         return static::randomElement(static::$doy);
     }
 
+    /**
+     * @example '111111111'
+     */
+    public static function grafm()
```

Comment:
Should 

- this method be called `vat()` 
- this method be located in `Faker\Provider\el_GR\Payment`

?

Reply:
Greek guy here 

Note that `vat` means the value added to a product.. Title is misleading as a Greek guy here this happens very often to me as well.

 `afm` though is the Tax Identification number of each Greek company/citizen

Back to our subject, well.. since two lines above there is `doy` which is the greeklish version of `Tax office` you could have `grafm` here as well which is the greeklish version of Tax Identification Number

The more accurate indeed would be to be named TIN and return in it's international form `EL123456789` but if you have a PHP application which does not read it in it's international form and reads it as `123456789` generating fake data for that app with the `tin` function will not help you..

Concluding I suggest to stay as is and maybe have later an enhancement adding tin functions to this package